notebooks/HASP/DataDiagnostic/DataDiagnostics.ipynb

```diff
@@ -0,0 +1,1599 @@
+{
```

Comment:
<p>It would be beneficial to expand the docstring to include more details.</p><p>Also the notebook uses a mix of .format and f-string for formatting print statements. I recommend converting all string formatting to use f-strings.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/hst_notebooks/pull/207/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>